Fortran/gfortran/regression/gomp/DisabledFiles.cmake

```diff
@@ -45,84 +35,49 @@ file(GLOB UNIMPLEMENTED_FILES CONFIGURE_DEPENDS
 
   # unimplemented: derived type components with non default lower bounds
   depend-iterator-1.f90
-  depend-iterator-2.f90
-
-  # unimplemented: assumed rank in procedure interface
-  pr94672.f90
```

Comment:
Not sure how this started working.

Reply:
This is expected to raise a compile-time error with the message 

> 'array' not specified in enclosing 'parallel'

In flang, this now errors with the message:

> The DEFAULT(NONE) clause requires that 'array' must be listed in a data-sharing attribute clause

Are these not equivalent? 

It was probably hitting a TODO earlier with the assumed rank error message.